packages/mask/src/UIRoot.tsx

```diff
@@ -33,7 +33,7 @@ function MaskThemeProvider({ children, baseline, useTheme }: MaskThemeProvider)
                 disableWindowBlurListener={false}
                 anchorOrigin={{ vertical: 'top', horizontal: 'right' }}
                 children={jsx}
-                isFacebook={activatedSocialNetworkUI.networkIdentifier === FACEBOOK_ID}
+                isfacebook={Boolean(isFacebook(activatedSocialNetworkUI)).toString()}
```

Comment:
Browser complains that this DOM attribute should be a string, attribute name non-camelcase.